src/cdp/transport.ts

```diff
@@ -108,10 +109,17 @@ export class WebSocketTransport implements ITransport {
   onmessage?: (message: string, receivedTime: bigint) => void;
   onend?: () => void;
 
-  static create(url: string, cancellationToken: CancellationToken): Promise<WebSocketTransport> {
+  static async create(
+    url: string,
+    cancellationToken: CancellationToken,
+  ): Promise<WebSocketTransport> {
+    const isSecure = !url.startsWith('ws://');
+    const targetAddressIsLoopback = await isLoopback(url);
+
     const ws = new WebSocket(url, [], {
       perMessageDeflate: false,
       maxPayload: 256 * 1024 * 1024, // 256Mb
+      rejectUnauthorized: !(isSecure && targetAddressIsLoopback),
```

Comment:
Shouldn't this be

```suggestion
      rejectUnauthorized: isSecure && targetAddressIsLoopback,
```

?